email_template_configurator/tests/test_configurator.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests import common
+
+
+class test_configurator(common.TransactionCase):
+
+    def test_configurator(self):
+        placeholders_obj = self.env['email.template.placeholder']
+        templates_obj = self.env['email.template']
+        invoice_model = self.env['ir.model'].search([
+            ('model', '=', 'account.invoice'),
```

Comment:
@andreparames 
Is this test will work properly if the "account" module is not loaded ?